src/utils/validateUser.js

```diff
@@ -28,14 +28,12 @@ exports.validateUpdateInput = Joi.object({
   firstName: Joi.string()
```

Comment:
Please, move this change to the separate merge request, it is not related to the tests